lib/ansible/modules/storage/purestorage/purefa_host.py

```diff
@@ -0,0 +1,207 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# (c) 2017, Simon Dodsley (simon@purestorage.com)
+#
+# This file is part of Ansible
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible. If not, see <http://www.gnu.org/licenses/>.
+
+ANSIBLE_METADATA = {'metadata_version': '1.0',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+
+DOCUMENTATION = '''
+---
+module: purefa_host
+version_added: 2.4
+short_description: Create, Delete and Modify Hosts on Pure Storage FlashArray
+description:
+    - This module creates, deletes or modifies hosts on Pure Storage FlashArray.
+author: Simon Dodsley (@simondodsley)
+options:
+  host:
+    description:
+      - Host Name
+    required: true
+  state:
+    description:
```

Comment:
For consistence please ensure all `description:` lines are full sentences (capital letters + full stop at the end)